tests/testPhotoCal.py

```diff
@@ -161,9 +194,12 @@ def testFlags(self):
             if source.get("calib_photometryUsed"):
                 used += 1
             if source.get("calib_photometryReserved"):
+                reservedSum -= source.getId()
                 reserved += 1
-        self.assertEqual(reserved, int(.3 * candidates))
+        self.assertEqual(reserved, int(.3 * (candidates + reserved)))
```

Comment:
The code was changed so that sources which are reserved are not marked as candidates.  I think I got a comment from somewhere saying that this was the correct semantics.  I will check before I end this review.